src/components/dex/MarketPairChart.vue

```diff
@@ -48,42 +48,44 @@
 </template>
 
 <script>
+import { mapGetters } from 'vuex';
 
 export default {
   computed: {
     currentMarket() {
       return this.$store.state.currentMarket ? this.$store.state.currentMarket.marketName : '';
     },
 
+    currentMarketData() {
+      return this.marketData[this.currentMarket];
+    },
+
+    currentTickerData() {
+      return this.tickerData[this.currentMarket];
+    },
+
     basePrice() {
       const tradeHistory = this.$store.state.tradeHistory[this.currentMarket];
       const hasTradeHistory = tradeHistory && tradeHistory.trades && tradeHistory.trades.length > 0;
       return this.$formatTokenAmount(hasTradeHistory ? tradeHistory.close24Hour : 0);
     },
-  },
 
-  props: {
-    baseCurrencyUnitPrice: {
-      default: 0,
-      type: Number,
+    quoteVolume() {
+      return _.get(this.tickerData, [this.currentMarket, 'quoteVolume'], 0)
```

Comment:
this is equivalent to:
this.currentTickerData() ? this.currentTickerData().quoteVolume : 0

quoteVolume should probably be a computed field though because it is being used here and in the place of my first comment of this review.